.env.development

```diff
@@ -17,6 +18,7 @@ ENTERPRISE_LEARNER_PORTAL_URL='http://localhost:8734'
 ENTERPRISE_SUPPORT_URL='https://edx.org'
 ENTERPRISE_SUPPORT_REVOKE_LICENSE_URL='https://edx.org'
 ENTERPRISE_SUPPORT_PROGRAM_OPTIMIZATION_URL='https://edx.org'
+ENTERPRISE_SUPPORT_LEARNER_CREDIT_URL='https://edx.org'
```

Comment:
As discussed, we are leaving a value defined in .env.development for now, at least until this file is fully deprecated and a replacement provided (dev.config.js???). This helps not completely break a misconfigured (by default) local development environment that hasn't enabled the config API.